recipes/onedpl/all/conanfile.py

```diff
@@ -54,12 +59,16 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
 
     def package_info(self):
-        self.cpp_info.filenames["cmake_find_package"] = "ParallelSTL"
-        self.cpp_info.filenames["cmake_find_package_multi"] = "ParallelSTL"
-        self.cpp_info.names["cmake_find_package"] = "pstl"
-        self.cpp_info.names["cmake_find_package_multi"] = "pstl"
-        self.cpp_info.components["_onedpl"].names["cmake_find_package"] = "ParallelSTL"
-        self.cpp_info.components["_onedpl"].names["cmake_find_package_multi"] = "ParallelSTL"
+        find_package_name = "ParallelSTL"
+        target_name = "pstl"
+        self.cpp_info.filenames["cmake_find_package"] = find_package_name
+        self.cpp_info.filenames["cmake_find_package_multi"] = find_package_name
+        self.cpp_info.names["cmake_find_package"] = target_name
+        self.cpp_info.names["cmake_find_package_multi"] = target_name
+        self.cpp_info.set_property("cmake_file_name", find_package_name)
+        self.cpp_info.set_property("cmake_target_name", target_name)
```

Comment:
I think this property now stores the absolute target name

Reply:
fixed in latest push

Reply:
That works locally but appears to fail on CI - 
```
  Syntax error in cmake code at

    /home/conan/w/BuildSingleReference/conan-center-index/recipes/onedpl/all/test_package/build/2da8b83836e27aacd43916b404a0e9d98f21239f/ParallelSTLTarget-release.cmake:95

  when parsing string

    ${pstl::ParallelSTL_FRAMEWORKS_RELEASE}
```